conans/client/file_copier.py

```diff
@@ -31,7 +32,7 @@ class FileCopier(object):
     imports: package folder -> user folder
     export: user folder -> store "export" folder
     """
-    def __init__(self, root_source_folder, root_destination_folder, excluded=None):
+    def __init__(self, root_source_folder, root_destination_folder):
```

Comment:
I think we should rename the `root_source_folder` to `root_folders` and force it to be always a list (changing the calls to it). Shouldn't be a problem, right? Or is this class public somehow?

Reply:
I have renamed it, but a bit ugly if it is always a list argument in the constructor, I think it could accept a single folder in the constructor and convert to a list if necessary (internally it is always a list)

Reply:
Done, forced a list always.